geode-core/src/main/java/org/apache/geode/Delta.java

```diff
@@ -32,7 +32,9 @@
 public interface Delta {
 
   /**
-   * Returns true if this object has pending changes it can write out.
+   * Returns true if this object has delta to be applied.
+   * If true, delta is applied and delta data is sent between client/server, otherwise entire
+   * data is sent between client/server.
```

Comment:
I'm sorry but this javadoc doesn't seem at all clear to me and it has grammatical errors.  I think you should keep the original Javadoc and append something like this:
_Returns false if pending changes are not available, which will cause the entire object to be transmitted to other processes._

Reply:
Thanks Bruce. I wanted to make it clear what is communicated based on hasDelta().

Reply:
Here's my version:
```
Returns true if this object has pending changes that can be transmitted as a delta.
```
If that's not enough then maybe:
```
Returns true if this object has pending changes that can be transmitted as a delta. Returns false if this object must be transmitted in its entirety.
```